### DISCUSSION TITLE
Trace after Laravel (Injecting header to requests)

### DISCUSSION THREAD

**Discussion by 090809** [spatie/laravel-open-telemetry#5] [Ideas]
Hey there! 
What about Injecting tracing carrier onto laravel HTTP requests via headers?

I think, it might be done with Http macro method.

**Comment by freekmurze** [comment]
Feel free to send a PR. Be sure to include tests and updated readme.

**Reply by 090809** [reply]
https://github.com/spatie/laravel-open-telemetry/pull/6
